src/main/java/it/eng/idsa/businesslogic/processor/common/RegisterTransactionToCHProcessor.java

```diff
@@ -32,23 +32,21 @@
 	@Override
 	public void process(Exchange exchange) throws Exception {
 		if (!isEnabledClearingHouse) {
-            exchange.getOut().setHeaders(exchange.getIn().getHeaders());
-            exchange.getOut().setBody(exchange.getIn().getBody());
             logger.info("CH registration not configured - continued with flow");
             return;
```

Comment:
Is this not necessary anymore because of the new version?

Reply:
It is removed, see - and marked in red. It means it is removed :)

Reply:
The question is why is it removed? :)

Reply:
With new Camel version, they removed concept of "in" and "out", now you have only Message, same for "in" and "out" and if you want to change it, you just get message form exchange, and manipulate with it.

Reply:
Ok, thanks. :)